app/Http/Controllers/Contact/PasswordStoreController.php

```
@@ -0,0 +1,14 @@
+<?php
+
+
+namespace App\Http\Controllers\Contact;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\PasswordRequest;
+
+class PasswordStoreController extends Controller
+{
+    public function __invoke(PasswordRequest $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)